src/components/Button/Base/Button.js

```diff
@@ -90,20 +90,17 @@ Button.propTypes = {
    * Optional additional className
    */
   className: PropTypes.string,
-  /**
```

Comment:
We have these because `Storybook` did not show enums at some point. Would make sense to remove all of these since Storybook4 seems to support it?